packages/api/src/libs/queries/emp.ts

```diff
@@ -210,7 +210,7 @@ export default (appState: Dependencies) => {
     latestPriceByTokenAddress,
     historicalPricesByTokenAddress,
     sliceHistoricalPricesByTokenAddress,
-    getGlobalTvl,
+    getTotalTvl,
```

Comment:
in the emp scoped queries, this is just renamed to total tvl, rather than global tvl, to indicate its total across all emps. reserving the global word to be across all contract types